pkg/triage/github.go

```diff
@@ -45,3 +47,11 @@ func MustReadToken(path string, env string) string {
 	}
 	return token
 }
+
+func MustCreateGithubClient(githubAPIRawURL string, httpClient *http.Client) (*github.Client, error) {
```

Comment:
This can be named just `GitHubClient`: `Must` as a prefix is reserved in go for functions which panic or otherwise exit on failure.

Reply:
I'd like to keep the name and change the method to exit when error.